test/lib/clients/oidc_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+# NOTE: these tests excercise the OIDC config but can't really verify a
+# successful OIDC login.  (Because that requires browser interaction.)
+# See the readme for how to use oidc login with the browser.
+
+class OIDCTest < ActiveSupport::TestCase
+  setup do
+    @client = Clients::Vault
+  end
+
+  test "#configure_oidc_user" do
```

Comment:
This seems like two tests worth of testing.

Reply:
done